go.mod

```diff
@@ -7,16 +7,15 @@ require (
 	github.com/dajohi/goemail v1.0.0
 	github.com/davecgh/go-spew v1.1.1
 	github.com/decred/dcrd/certgen v1.0.2
-	github.com/decred/dcrd/chaincfg v1.4.0
+	github.com/decred/dcrd/chaincfg v1.5.1
 	github.com/decred/dcrd/chaincfg/chainhash v1.0.1
 	github.com/decred/dcrd/dcrec/secp256k1 v1.0.1
-	github.com/decred/dcrd/dcrutil v1.2.1-0.20190118223730-3a5281156b73
+	github.com/decred/dcrd/dcrutil v1.3.0
```

Comment:
`go` forced these other updates when updating the dcrdata ones.  Not sure why, the changes seem to be non-breaking.  We updated to these dcrd modules for dcrdata 5.0